apps/api/src/app/bridge/bridge.controller.ts

```diff
@@ -214,17 +203,22 @@ export class BridgeController {
 
   @Post('/validate')
   @ExternalApiAccessible()
-  async validateBridgeUrl(@Body() body: ValidateBridgeUrlRequestDto): Promise<ValidateBridgeUrlResponseDto> {
+  @UseGuards(UserAuthGuard)
+  async validateBridgeUrl(
+    @UserSession() user: UserSessionData,
+    @Body() body: ValidateBridgeUrlRequestDto
+  ): Promise<ValidateBridgeUrlResponseDto> {
     try {
-      const result = await this.validateBridgeUrlUsecase.execute(
+      const result = await this.getBridgeStatus.execute(
         GetBridgeStatusCommand.create({
-          bridgeUrl: body.bridgeUrl,
+          environmentId: user.environmentId,
+          statelessBridgeUrl: body.bridgeUrl,
         })
       );
 
       return { isValid: result.status === 'ok' };
     } catch (err: any) {
-      return { isValid: false };
+      return { isValid: false, error: err.message };
```

Comment:
Actual error is returned to client now to make debugging easier.

Reply:
Good idea, I've added log.error logging for all Bridge execution errors for the time-being to make debugging easier for ourselves.